community.adoc

```diff
@@ -4,10 +4,27 @@ title: Community
 permalink: /community/
 ---
 
-## Welcome
+# Welcome to the Kaskada community!
+
+The Kaskada project was open-sourced in 2023. As a growing community, we're here to explore new ways to interact with data at scale and have some fun along the way. We build in the open and use the channels below for communication.
 
 ## Discussions
 
-## Events
+For news and announcements, follow us on Twitter: https://twitter.com/kaskadainc[@kaskadainc].
+
+Join the Stargate group on LinkedIn: https://www.linkedin.com/company/kaskadainc[@kaskadainc].
+
+
+## Contributing
+
+Nothing makes us happier than community contributions. See the https://github.com/kaskada-ai/kaskada/blob/main/README.md[README] in the main Kaskada repository to get a grasp of the project layout and reference the https://github.com/kaskada-ai/kaskada/blob/main/CONTRIBUTING.md[CONTRIBUTING] file for detailed steps.
```

Comment:
Suggestion for adoc -- create a placehoder for the repo. I did this:

```
:repo: https://github.com/riptano/kaskada

...

link:{repo}/issues[issues]
```

It lets us have a placehoder that gets substited in for the repo. We may be able to just pull it from the site configuration.

This would let us more easily move things in the future.